sentences/nl/fan_HassTurnOn.yaml

```diff
@@ -3,18 +3,18 @@ intents:
   HassTurnOn:
     data:
       - sentences:
-          - "[<doe>] [alle] <ventilator> [<naar>] aan in <area>"
-          - "Schakel [alle] <ventilator> [<naar>] in in <area>"
-          - "[<doe>] [alle] <ventilator> [in|op] <area> [<naar>] aan"
+          - "[<doe>] [alle] <ventilator> [<naar>] aan [willen | kunnen] [<doe>] in <area>"
+          - "Schakel [alle] <ventilator> [<naar>] in <area>"
```

Comment:
The double `in` was correct here, please leave it in
`Schakel alle ventilatoren in in de woonkamer`